### ISSUE TITLE
Cannot open xplorer on Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by george-martinec** [kimlimjustin/xplorer#264]
### Description

Error:
xplorer: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory

### Steps To Reproduce

1. sudo dpkg -i xplorer_0.3.1_amd64.deb
2. sudo xplorer

### Expected behavior

Open xplorer

### Xplorer Version

v0.3.1

### Operating System Version

Ubuntu 22.04 LTS

**Comment by george-martinec** [comment]
Fixed by this article: https://fedingo.com/how-to-install-openssl-in-ubuntu/

**Comment by devoidfury** [comment]
Well, installing an old version of openssl (1.1.1) instead of the modern versions shipping (>3) isn't really a "fix". Instead, it'd be better to update the dependencies.

**Comment by matbgn** [comment]
Let me double this it's really NOT a fix it's a security issue moreover. Any plan to update dependencies?

**Comment by kimlimjustin** [comment]
> Let me double this it's really NOT a fix it's a security issue moreover. Any plan to update dependencies?

I'm really sorry I got no linux to test it currently, but is there any idea to what version should I dump?

**Comment by matbgn** [comment]
As previously said, more than 3. ATTW 3.0.8 http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/?C=M;O=D

**Comment by matbgn** [comment]
> > Let me double this it's really NOT a fix it's a security issue moreover. Any plan to update dependencies?
> 
> I'm really sorry I got no linux to test it currently, but is there any idea to what version should I dump?

Sidenote: As it costs nothing, maybe it could be useful for you @kimlimjustin  as a dev in the future https://ubuntu.com/tutorials/how-to-run-ubuntu-desktop-on-a-virtual-machine-using-virtualbox#1-overview

**Comment by kimlimjustin** [comment]
> Sidenote: As it costs nothing, maybe it could be useful for you @kimlimjustin as a dev in the future https://ubuntu.com/tutorials/how-to-run-ubuntu-desktop-on-a-virtual-machine-using-virtualbox#1-overview

Yeah, but there is something wrong with my virtualbox, don't have time to fix it yet. Btw, if you don't mind, a PR would be great! (cuz I'm not free to code rn)

**Comment by matbgn** [comment]
I'm a newbie with Tauri so right now not really helpful but this will change in the coming months hopefully.

**Comment by kimlimjustin** [comment]
Okay then, I'll fix this together with the refactoring of the whole project in June (hopefully)

**Comment by MaloneQQ** [comment]
Ubuntu 22.04.5 LTS

wget https://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.24_amd64.deb
sudo dpkg -i libssl1.1_1.1.1f-1ubuntu2.24_amd64.deb

https://github.com/kimlimjustin/xplorer/issues/296
